frontend/src/pages/NewRunV2.tsx

```diff
@@ -522,6 +536,19 @@ function NewRunV2(props: NewRunV2Props) {
                 }
               }}
             />
+            <FormControlLabel
```

Comment:
This form is also used for one off runs, so it'd be helpful to disable this check box if "Recurring" isn't selected.